### PR TITLE
man: document differences in clean exit status for Type=oneshot

### PR DESCRIPTION
**Pull request by dtardon**

### CODE REVIEW

**Comment by bluca** [comment]
Could you please link some references in the commit message? I don't see an exception in here: https://github.com/systemd/systemd/blob/main/src/shared/exit-status.c#L149 so it would be good to leave a crumb trail

File: man/systemd.service.xml [hunk 1]
```diff
@@ -756,14 +756,19 @@
...
-        <varname>SuccessExitStatus=</varname>. If set to
+        In this context, a clean exit means any of the following:
+        <itemizedlist>
+            <listitem><simpara>Exit code of 0.</simpara></listitem>
+            <listitem><simpara>One of the signals
+                <constant>SIGHUP</constant>,
+                <constant>SIGINT</constant>,
+                <constant>SIGTERM</constant> or
+                <constant>SIGPIPE</constant> (except for
+                <varname>Type=oneshot</varname>).</simpara></listitem>
```
Comment (keszybz) [inline man/systemd.service.xml]: Oxford comma please.
It is not clear if the added parenthetical remark applies to SIGPIPE or all. Actually I assumed the former, and had to look at the code to understand this. Maybe reword this as
> For types other than Type=oneshot, one of the signals ...

Also, please use ";" instead of "." at the end of the items.

**Review by keszybz — CHANGES_REQUESTED** [review]

File: man/systemd.service.xml (hunk 1 above)
Comment (dtardon) [reply man/systemd.service.xml]: Updated.

**Comment by keszybz** [comment]
Thanks.
